curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which browser is known for robust developer tools?
 
 #### --distractors--
 
-Placeholder distractor 1
+Safari
 
 ---
 
-Placeholder distractor 2
+Internet Explorer
 
 ---
 
-Placeholder distractor 3
+Opera
 
 #### --answer--
 
-Placeholder answer
+Firefox
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a common use for aria-hidden="true"?
 
 #### --distractors--
 
-Placeholder distractor 1
+To show content
 
 ---
 
-Placeholder distractor 2
+To display on mobile
 
 ---
 
-Placeholder distractor 3
+To increase visibility
 
 #### --answer--
 
-Placeholder answer
+To hide content from screen readers
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the flex property used for?
 
 #### --distractors--
 
-Placeholder distractor 1
+Changing text color
 
 ---
 
-Placeholder distractor 2
+Adjusting font size
 
 ---
 
-Placeholder distractor 3
+Adding animations
 
 #### --answer--
 
-Placeholder answer
+Creating responsive layouts
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which tool allows you to test CSS live on a website?
 
 #### --distractors--
 
-Placeholder distractor 1
+CodePen
 
 ---
 
-Placeholder distractor 2
+JSFiddle
 
 ---
 
-Placeholder distractor 3
+Live CSS Editor
 
 #### --answer--
 
-Placeholder answer
+All of the above
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which attribute is essential for accessible images?
 
 #### --distractors--
 
-Placeholder distractor 1
+src
 
 ---
 
-Placeholder distractor 2
+title
 
 ---
-
-Placeholder distractor 3
+width
 
 #### --answer--
 
-Placeholder answer
+alt
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the minimum contrast ratio for graphics?
 
 #### --distractors--
 
-Placeholder distractor 1
+1.5:1
 
 ---
 
-Placeholder distractor 2
+4.5:1
 
 ---
 
-Placeholder distractor 3
+3:1
 
 #### --answer--
 
-Placeholder answer
+7:1
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which feature of Chrome helps in accessibility audits?
 
 #### --distractors--
 
-Placeholder distractor 1
+DevTools
 
 ---
 
-Placeholder distractor 2
+Extensions
 
 ---
 
-Placeholder distractor 3
+Bookmark Manager
 
 #### --answer--
 
-Placeholder answer
+Lighthouse
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which property aligns text in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+align-text
 
 ---
 
-Placeholder distractor 2
+justify-text
 
 ---
 
-Placeholder distractor 3
+text-position
 
 #### --answer--
 
-Placeholder answer
+text-align
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the outline property affect?
 
 #### --distractors--
 
-Placeholder distractor 1
+Border of an element
 
 ---
 
-Placeholder distractor 2
+Margin of an element
 
 ---
 
-Placeholder distractor 3
+Padding of an element
 
 #### --answer--
 
-Placeholder answer
+Focus ring around an element
```

Comment:
Outlines don't immediately affect focus indicators. Rather they affect the border of an element. You can use it in a pseudo-class to change a focus indicator, but it is beyond the scope of this question. 

Source: https://developer.mozilla.org/en-US/docs/Web/CSS/outline